exercises/rotational-cipher/rotational_cipher.exs

```diff
@@ -0,0 +1,13 @@
+defmodule RotationalCipher do
+  @doc """
+  Given a plaintext and amount to shift by, return a rotated string.
+
+  Example:
+  iex> RotationalCipher.rotate("Attack at dawn", 13)
+  "Nggnpx ng qnja"
+  """
+  @spec rotate(text :: String.t(), shift :: integer) :: String.t()
+  def rotate(_text, _shift) do
```

Comment:
I don't like it either, but I didn't feel strongly enough about it to warrant asking for a change. If it doesn't mess with CI, then I'd be happy to see them gone.